docs/bib/HEPData_likelihoods.bib

```diff
@@ -61,6 +61,15 @@ @misc{ins1827025
     collaboration = "ATLAS",
 }
 
+@misc{ins1802524,
```

Comment:
This is placed here to try to place it correctly in the chronology based on the "24 June 2020" on the [top group public results page](https://atlas.web.cern.ch/Atlas/GROUPS/PHYSICS/PAPERS/TOPQ-2020-02/) and the "Last updated on 2020-07-28 13:51" on the [HEPData page](https://doi.org/10.17182/hepdata.95748).